lib/Rfc1035StubResolver.php

```diff
@@ -494,4 +512,19 @@ private function assertAcceptableResponse(Message $response)
             throw new DnsException(\sprintf("Server returned error code: %d", $response->getResponseCode()));
         }
     }
+
+    private function getNameserver(int $attempt = 0): string
```

Comment:
This should be named `selectNameserver` or similar, getters should be side-effect free.

Reply:
@kelunik I was thinking about putting "next" but like the `selectNameserver` as well.